typescript/cli/src/utils/aws.ts

```diff
@@ -0,0 +1,28 @@
+import { input } from '@inquirer/prompts';
+
+export async function getAccessKeyId(skipConfirmation: boolean) {
```

Comment:
Please move all this code to typescript/cli/src/validator/address.ts at the bottom